src/services/event.js

```diff
@@ -273,11 +277,83 @@ const makeMatchesFilters = (filters) => (event) => {
   return false;
 };
 
+/**
+ * Splits up multiple occurrences of events into individual events
+ * and re-sorts the array
+ *
+ * @param {Object[]} events the array of events to process
+ * @returns {Object[]} the flattened array of events after splitting multiple occurrences
+*/
+const processMultipleOccurences = (events) => {
```

Comment:
I think that instead of having this as a separate function called by all the HTTP functions, you could make this a helper function of `formatEvent` and tweak it just slightly to process occurrences for a single event. That way, the processing is included in the function that is already used to format the events in all other ways.

Reply:
I'm not sure that is possible because formatEvent is supposed to return one event. This function would still have to return multiple events (for each occurrence). Thoughts?

Reply:
Hmm, that's a good point.

There is a solution, but it would mean returning an array of events from `formatEvent` and then `flatmap`ping over `formatEvent` instead of `map`ping. That would work perfectly, but it might be opaque to people who aren't as familiar with the process.

Reply:
I'm realizing now that, in reality, we should probably be doing this work in the backend (this is prime backend kind of data normalization). Especially because processing large amounts of data in a uniform way is not something that end user devices are good at.

Reply:
Hmmm. So you think hold off on this one then @EjPlatzer ?